### PR TITLE
Fixed skeleton view

### PR DESCRIPTION
**Pull request by ulteq**
- [x] ~~Fix skeleton view overlapping everything like player, models, GUI~~
- [x] Fix visual glitch when the skeleton view is active on multiple trucks of the same kind
- [x] Fix skeleton view propagation

Fixes: https://github.com/RigsOfRods/rigs-of-rods/issues/1615

### CODE REVIEW

**Comment by ulteq** [comment]
@only-a-ptr the last issue is probably hard to fix.

**Comment by only-a-ptr** [comment]
@ulteq The overlaping is a result of using DearIMGUI to draw the skeletonview in screen space. Also, without the overlap, vehicle's own meshes and cabs would obscure the view, no matter what trick you use to fade them. I like the overlapping as a feature - it's how I'd expect a diagnostic view to behave.

**Comment by ulteq** [comment]
> I like the overlapping as a feature - it's how I'd expect a diagnostic view to behave.

Right, so we can merge it?

File: source/main/gfx/GfxActor.cpp [hunk 1]
```diff
@@ -697,7 +697,8 @@ void RoR::GfxActor::UpdateDebugView()
     // Dummy fullscreen window to draw to
     int window_flags = ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoScrollbar| ImGuiWindowFlags_NoInputs 
                      | ImGuiWindowFlags_NoSavedSettings | ImGuiWindowFlags_NoFocusOnAppearing | ImGuiWindowFlags_NoBringToFrontOnFocus;
-    ImGui::Begin("RoR-SoftBodyView", NULL, screen_size, 0, window_flags);
+    static unsigned int counter = 0;
+    ImGui::Begin(("RoR-SoftBodyView-" + TOSTRING(counter++)).c_str(), NULL, screen_size, 0, window_flags);
```
[the remark this answers is not in the document]
Comment (ulteq) [reply source/main/gfx/GfxActor.cpp]: > Use actor ID

Good idea.

**Pull request merged**